packages/forms/resources/css/components/file-upload.css

```diff
@@ -5,6 +5,7 @@
 @import 'filepond-plugin-media-preview/dist/filepond-plugin-media-preview.css';
 
 .filepond--root {
+    font-family: var(--font-family);
```

Comment:
@danharrin @urufudev, can't we just use `font-sans` in `@apply` instead?

Reply:
These files are not compiled with Tailwind.

Reply:
Hmm fair, but then the font family won't work outside Filament panels...

Reply:
It will fall back to the Filepond default, or they can set the CSS variable. Or they can add the class to their own project.

Or we can try wrapping Filepond in an `!font-sans` in the HTML and hope that `!important` overrides this?

Reply:
If that doesn't work, we could use `[.filepond--root]-font-sans`?

Reply:
Just like I've done here: https://github.com/filamentphp/filament/blob/932007dbdd36e6b7765eb525df0c448561229a2e/packages/forms/resources/views/components/select.blade.php#L137.

Reply:
Sure, can you change that then please?

Reply:
Actually, these files will be compiled with Tailwind, @danharrin?
There's literally a lot of `@apply` statements in here. 😂
I think we should simply be able to use `font-sans`.
Correct me if I'm wrong though.

Reply:
The end user's Tailwind config is not used, it's our own

Reply:
Ah of course.

Reply:
Fixed by https://github.com/filamentphp/filament/commit/5ad3677b4b7ba24702f9b3ba3c0851d7e466fb10